docs/installation/others/creating-ds-instance.adoc

```diff
@@ -18,57 +19,64 @@ _If you wish to disable bootstrap certificate generation and SSL connection, set
 
 == Creating a DS Instance 
 
-=== Generate a DS configuration file, for example `ds.inf`: 
+=== Generating a DS configuration file, for example `ds.inf`: 
 
-----
+[literal,subs="+quotes,verbatim"]
+....
 $ dscreate create-template ds.inf
-----
+....
 
 Customize the DS configuration file as follows:
 
-----
+[literal,subs="+quotes,verbatim"]
+....
 $ sed -i \
     -e "s/;instance_name = .*/instance_name = localhost/g" \
     -e "s/;root_password = .*/root_password = Secret.123/g" \
     -e "s/;suffix = .*/suffix = dc=example,dc=com/g" \
     -e "s/;create_suffix_entry = .*/create_suffix_entry = True/g" \
     -e "s/;self_sign_cert = .*/self_sign_cert = True/g" \
     ds.inf
-----
+....
 
 where
 
 * *instance_name* specifies the name of the DS instance. In this example it's set to `localhost`.
-* *root_password* specifies the password for DS admin (i.e. `cn=Directory Manager`). In this example it's set to `Secret.123`.
-* *suffix* specifies the namespace for the DS instance. In this example it's set to `dc=example,dc=com`.
-* *self_sign_cert* specifies whether to create self-signed certificates for SSL connection. In this example it's set to `True`. The SSL connection is enabled.
+
+* *root_password* specifies the password for DS admin, that is `cn=Directory Manager`. In this example it is set to `Secret.123`.
```

Comment:
not sure I understand why blank lines are needed between these bullets.  But I'll leave them be for this PR for now.